data/fields/traffic_signals/sound.json

```diff
@@ -0,0 +1,21 @@
+{
+    "key": "traffic_signals:sound",
+    "type": "combo",
+    "label": "Sound Signals",
+    "strings": {
+        "options": {
+            "yes": "Yes",
+            "no": "No",
+            "locate": {
+                "title": "Locate",
+                "description": "There is only a signal to find the pole"
+            },
+            "walk": {
+                "title": "Locate",
```

Comment:
`locate` and `walk` both have the same `title` of “Locate”. I think this would be “Walk”, but “Walk Indication” would be consistent with “Call Button Locator” above and [American English terminology](http://www.apsguide.org/chapter4_walkindication.cfm).